database/views.py

```diff
@@ -5,7 +5,6 @@
 from database.models import *
 import json
 import requests
-
```

Comment:
Can we have the `permission_classes = [IsAuthenticated]` line in the views rather than the models? Or does it not make a difference?  Also, we need to check if the user is an admin for the corresponding get requests

Reply:
An example of this is here https://github.com/leavescalculator/LeavesCalculator/blob/1f237c5ddd2427d614c6d8f7bfba6f8bff0efd3e/calculator/api/views.py#L31

Reply:
I think that is correct, that's certainly where we had it before